### ISSUE TITLE
Lables

### ISSUE THREAD

**Issue by lionelxia** [RuudFirsa/Smiles-GEN#2]
I can't understand what's labels.
Can you give me an example?
'The generated text corpus was converted to a training set pairing the next  available characters (labels) to the previously observed sentence, which are presented as one-hot encoded feature matrices to the network.'
The above sentence is what I saw in the paper.
